File: dataModuleTester/src/main/java/com/aneonex/bitcoinchecker/tester/volley/CheckerVolleyNextRequest.java
package com.aneonex.bitcoinchecker.tester.volley;

import java.util.Map;

import com.android.volley.toolbox.RequestFuture;
import com.aneonex.bitcoinchecker.datamodule.model.CheckerInfo;
import com.aneonex.bitcoinchecker.tester.volley.generic.GenericCheckerVolleyRequest;

public class CheckerVolleyNextRequest extends GenericCheckerVolleyRequest<String> {
	
	public CheckerVolleyNextRequest(String url, CheckerInfo checkerInfo, RequestFuture<String> future) {
		super(url, checkerInfo, future, future);
	}

	@Override
	protected String parseNetworkResponse(Map<String, String> headers, String responseString) throws Exception {
		return responseString;
	}
}

File: dataModuleTester/src/main/java/com/aneonex/bitcoinchecker/tester/volley/CheckerVolleyNextRequest.kt
package com.aneonex.bitcoinchecker.tester.volley

import com.android.volley.toolbox.RequestFuture
import com.aneonex.bitcoinchecker.datamodule.model.CheckerInfo
import com.aneonex.bitcoinchecker.datamodule.model.PostRequestInfo
import com.aneonex.bitcoinchecker.tester.volley.generic.GenericCheckerVolleyRequest

class CheckerVolleyNextRequest(url: String?, postRequestInfo: PostRequestInfo?, checkerInfo: CheckerInfo, future: RequestFuture<String>)
    : GenericCheckerVolleyRequest<String?>(url, postRequestInfo, checkerInfo, future, future) {
    @Throws(Exception::class)
    override fun parseNetworkResponse(headers: Map<String?, String?>?, responseString: String?): String? {
        return responseString
    }
}